rktl_game/web/index.js

```diff
@@ -0,0 +1,79 @@
+// Connecting to ROS
+// -----------------
+
+var ros = new ROSLIB.Ros({
+    url: 'ws://r2d2.local:9090'
```

Comment:
Is there a way to make this run on any computer? Like `localhost:9090`?

Reply:
localhost did not work with `--net host` or `-p 9090`

Reply:
and if we want it to be viewed on a different device (like a phone) `localhost` wouldn't work